.circleci/config.yml

```diff
@@ -58,6 +58,14 @@ commands:
             google-chrome --version
             chromedriver --version
           name: Check install
+  install_js_dependencies:
+    description: 'Install JavaScript dependencies'
+    steps:
+      - run: yarn install
```

Comment:
Typically you want to use `yarn install --frozen-lockfile` in CI to ensure no one forgets to check-in lockfile changes.